js/ui/number_box/number_box.mask.js

```diff
@@ -302,7 +302,6 @@ const NumberBoxMask = NumberBoxBase.inherit({
             }
         }
 
-        text = this._removeStubs(text, true);
```

Comment:
seems like this function expects full text with stubs as a parameter. Are u sure that 298 row works correctly now? I think that `text.length` can return unexpected value now

Reply:
No tests are failed. May be, do you have a sample that demonstrate incorrect case?

Reply:
this code was added for T972648, please test scenarios from this ticket

Reply:
https://teams.microsoft.com/l/message/19:82ff4bea5a9f44659092e290f5f90bbe@thread.skype/1613164258551?tenantId=e4d60396-9352-4ae8-b84c-e69244584fa4&groupId=8c31b3e5-5870-46b1-92a7-180d13b282ed&parentMessageId=1613164258551&teamName=DevExtreme&channelName=Editors%20Squad&createdTime=1613164258551

Reply:
I checked these scenarios. They work as expected.